user-guide/howto/files/pxe/menu.ipxe

```diff
@@ -0,0 +1,46 @@
+#!ipxe
+
+set platform-version 20170624T192838Z
+set install-host 10.10.0.33
+
+######## MAIN MENU ###################
+:start
+menu Welcome to iPXE's Boot Menu
+item
+item --gap -- ------------------------- Operating systems ------------------------------
+item esdc_b1    Danube Cloud Compute Node (${platform-version})
+item esdc_r1    Danube Cloud Compute Node (${platform-version}) Rescue (l/p: root/root)
+item
+item --gap -- ------------------------------ Utilities ---------------------------------
+item shell      Enter iPXE shell
+item reboot     Reboot
+item
+item exit       Exit (boot local disk)
+choose --default esdc_b1 --timeout 30000 target && goto ${target}
```

Comment:
timeout of 30s is too long.. I suggest max 10s, maybe less 
(this timeout is used at every reboot, not only at first one)